Cargo.toml

```diff
@@ -26,7 +26,7 @@ futures = "0.3"
 hmac = "0.12"
 log = "0.4"
 rand = "0.8"
-rsa = "0.9"
+rsa = "=0.9.6"
```

Comment:
I would recommend not pinning to `0.9.6` to make sure this issue doesn't occur in other code paths, given that you have a workaround now